### ISSUE TITLE
Skill projectile -> Dmg type raycast no dmg

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#2658]
it works in singleplayer but in mmo it does no damage

**Comment by insthync** [comment]
Want to see your settings

**Comment by moepi2k** [comment]
actually just a skill and changed damageType to raycast. but i just noticed i have that problem for many skills in my game since updated to 191. nothing changed on the skills. in singleplayer all good in mmo i can cast the spell, effect projectile everythings here but just not apply damage. really a weird behaviour. i even can copy pase an existing skill which is working and then it does no damage. i thought its a broken meta thing. i removed library folder tried everything. even set debug logs and all logs are fine. i dont know what i should do

**Comment by insthync** [comment]
all logs are fine, did you log damage aaplying functions in damageable hitbox/entity?

**Comment by moepi2k** [comment]
yes debug damageable hitbox apply dmg. one thing i already figured out is projectiles which have speed of 60+ no longer do any dmg.

for now changed them to 50 and it works. but could be that other ppl will report this in future too.
now i need to check why my melee skill does no dmg which is actually just a simple skill which do melee dmg

https://gyazo.com/8af6ca3e79d51b2ed2bfeaed7a85898a

**Comment by insthync** [comment]
so it is projectile or raycast?

**Comment by moepi2k** [comment]
actually i have problems with all 3, missile, raycast, and melee. really annyoing thing which i struggle for days now. copy paste and same settings like other skills and one is working and the other not..

**Comment by insthync** [comment]
if it is raycast then it will raycast and perform hit reg immediately, if it is projectile, it will do it later when hit enemy, and this one is what you devloped and sent to me isn't it, I think I will remove it if it is not working

**Comment by moepi2k** [comment]
what you wanna remove? the thing is all my old skills worked fine, but since i update. i have random some which are not working, it doenst matter if missile, raycast or melee. 

need more time to check it

**Comment by insthync** [comment]
projectile damage entity

**Comment by moepi2k** [comment]
no i dont think there is any problem with projectile damage entity. i wouldnt remove it. i know alot of ppl using it because its really good. as said its not just missile problem in my project. its also having problems that some skills with melee dont do any dmg anymore. weird but i need to check it
